pyproject.toml

```diff
@@ -11,7 +11,7 @@ authors = [
 ]
 description = "Use this module to try out multiple interpreters and a per-interpreter GIL in Python 3.12.  Do not use this for anything important."
 readme = "README.md"
-requires-python = "=3.12"
+requires-python = ">=3.12,<3.13"
```

Comment:
I'm not a fan of `<`, especially for `requires-python`, but this is a valid Specifier, while `=` is not.